src/ui_utils.h

```diff
@@ -51,6 +51,8 @@ typedef struct GeanyInterfacePrefs
 	gchar			*tagbar_font;				/**< symbol sidebar font */
 	gchar			*msgwin_font;				/**< message window font */
 	gboolean		show_notebook_tabs;			/**< whether editor tabs are visible */
+	gboolean		auto_sort_tabs_filename;	/**< whether to automatically sort tabs based on filename */
+	gboolean		auto_sort_tabs_pathname;	/**< whether to automatically sort tabs based on pathname */
```

Comment:
> and it would be better to have one single setting that selects the mode (NONE/BASENAME/PATH) instead of two mutually exclusive booleans.

Agree.